app/kubemci/pkg/gcp/loadbalancer/loadbalancersyncer.go

```diff
@@ -219,6 +220,29 @@ func (l *LoadBalancerSyncer) PrintStatus() (string, error) {
 	return fmt.Sprintf("Load balancer %s has IPAddress %s and is spread across %d clusters (%s)", l.lbName, sd.IPAddress, len(sd.Clusters), strings.Join(sd.Clusters, ",")), nil
 }
 
+func formatForwardingRules(rules []status.LoadBalancerStatus) (string, error) {
+	// TODO: Should reuse printers in kubernetes/pkg/printers/printers.go
+	var result string = "List of multicluster loadbalancers created:\nName; IP; Clusters\n"
```

Comment:
call it multicluster ingresses instead of multicluster loadbalancers (since we call them mci's everywhere)?